Python/fileutils.c

```diff
@@ -2878,9 +2878,9 @@ _Py_GetLocaleconvNumeric(struct lconv *lc,
  *    non-opened fd in the middle.
  * 2b. If fdwalk(3) isn't available, just do a plain close(2) loop.
  */
-#ifdef __FreeBSD__
+#ifdef HAVE_CLOSEFROM
 #  define USE_CLOSEFROM
-#endif /* __FreeBSD__ */
+#endif /* HAVE_CLOSEFROM */
```

Comment:
Another option is to remove the `__FreeBSD__` guard and `s/USE_CLOSEFROM/HAVE_CLOSEFROM/g` in this file. This is a smaller diff, though. I'm fine with either.

Reply:
Yeah, fair point. I kept it this way for consistency with the `fdwalk` check, but if desired, I can change it. My preference is probably to keep it as-is though in the absence of anyone feeling strongly.

Reply:
This is fine.